src/platform/forms/tests/validations.unit.spec.js

```diff
@@ -479,12 +479,11 @@ describe('Validations unit tests', () => {
       expect(isValidPartialMonthYearInPast('2', '2001')).to.be.true;
     });
     it('should validate month and year that is current', () => {
+      const currentMonthIndexedAtOne = moment().month() + 1;
```

Comment:
Is possible for this to become > 12? If so, does it make sense to have values > 12 here? Or does the Valid Partial Month year automagically take into account month values > 12?

Reply:
Good question, but I don't know. My impression was that indexing starts at 1 for that helper.

Reply:
Based on how it subtracts a `1` from the month before passing it to the moment helper, I don't think it allows values greater than 12.

```
month: month ? parseInt(month, 10) - 1 : null,
```